.travis.yml

```
@@ -10,4 +10,4 @@ notifications:
 sudo: false
 script:
   - env -i PATH="$PATH" tests/test
-  - shasum --algorithm 1 --check --strict ./SHA1SUM
+  - for i in 1 256 512; do shasum --algorithm "${i}" --check --strict "./SHA${i}SUM"; done
```

Prefer this to be split out so it will fail if one is incorrect and easier to debug

```suggestion
  - shasum --algorithm 1 --check --strict ./SHA1SUM
  - shasum --algorithm 256 --check --strict ./SHA256SUM
  - shasum --algorithm 512 --check --strict ./SHA512SUM
```